python/setup.py

```diff
@@ -18,6 +18,7 @@
 # under the License.
 
 import contextlib
+import numpy
```

Comment:
It would be nice if NumPy was not necessary simply to run this file. Perhaps replace this with a lazy import in `build_extensions()` below?